### DISCUSSION TITLE
Github Education Verification

### DISCUSSION THREAD

**Discussion by TheShiningBoots** [community/community#139203] [GitHub Education]
### Hi everyone,

I had signed up for github education the previous year when I still had my school provided email ID, I was able to get verified then but recently our school withdrew all the accounts and as a result I am unable to re-register for the program since the edu email seems to now be compulsory, I have fee bills, ID cards, class schedules and admit cards for various examination just not the edu email since my school no longer gives us those. Is there any way for me to verify myself.

Also regarding my previous registration it expired since I forgot to renew it.

**Comment by alx-mp4** [comment]
Hi @TheShiningBoots, I recommend opening a ticket with GitHub support for this issue.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!
